client/src/components/icons/FoodIcon.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+
+export default function FoodIcon({ active }) {
+  return (
+    <svg
+      version="1.1"
+      id="Layer_1"
+      xmlns="http://www.w3.org/2000/svg"
+      xmlnsXlink="http://www.w3.org/1999/xlink"
+      x="0px"
+      y="0px"
+      className={active ? "active" : null}
```

Comment:
There's a really great package called `classnames` that allows for logic like this and makes it really easy to build on.  We don't necessarily need it here, but something to consider for the future.